creator-node/src/index.js

```diff
@@ -55,6 +56,13 @@ const startApp = async () => {
   if (!delegateOwnerWallet || !delegatePrivateKey || !creatorNodeEndpoint) {
     exitWithError('Cannot startup without delegateOwnerWallet, delegatePrivateKey, and creatorNodeEndpoint')
   }
+
+  // fail if delegateOwnerWallet doesn't derive from delegatePrivateKey
+  const privateKeyBuffer = Buffer.from(config.get('delegatePrivateKey').replace('0x', ''), 'hex')
+  const walletAddress = EthereumWallet.fromPrivateKey(privateKeyBuffer).getAddressString()
+  if (walletAddress !== config.get('delegateOwnerWallet').toLowerCase()) {
+    throw new Error('Invalid delegatePrivateKey/delegateOwnerWallet pair')
+  }
```

Comment:
looks good, curious why not use `web3.eth.accounts.privateKeyToAccount` (https://web3js.readthedocs.io/en/v1.3.0/web3-eth-accounts.html#privatekeytoaccount)

Reply:
this is how identity relay does key extraction, so just used the same code honestly

Reply:
ye makes sense. clearly it works